src/xcode/ENA/ENAUITests/TraceLocations/ENAUITests_10_TraceLocations.swift

```diff
@@ -56,4 +63,42 @@ class ENAUITests_10_TraceLocations: XCTestCase {
 		XCTAssertFalse(app.buttons[AccessibilityIdentifiers.General.primaryFooterButton].exists)
 	}
 
+	func test_screenshot_traceLocation_print_flow() throws {
+		app.launch()
+
+		// navigate to tracelocation card
+		app.swipeUp(velocity: .fast)
+		
+		// check if the tracelocation card exists
+		XCTAssertTrue(app.buttons[AccessibilityIdentifiers.Home.traceLocationsCardButton].waitForExistence(timeout: .short))
+		
+		// take snapshot
+		snapshot("tracelocation_card_home")
+
+		// navigate to tracelocation overview
+		app.buttons[AccessibilityIdentifiers.Home.traceLocationsCardButton].tap()
+
+		// take snapshot
+		snapshot("tracelocation_overview")
+
+		// navigate to tracelocation detail view for second item
+		app.cells.element(boundBy: 2).tap()
```

Comment:
This is very error prone. `app.cells` could mean ANY cell. Please define identifiers for the cells OR the `tableView` which then can be specified as `app.tables[$identifier].cells.element(boundBy: 2)`

Reply:
Fixed